src/lib/components/Identifiers/PIDField.js

```diff
@@ -301,25 +301,19 @@ class UnmanagedIdentifierCmp extends Component {
   render() {
     const { localIdentifier } = this.state;
     const { form, fieldPath, helpText, pidPlaceholder } = this.props;
-
+    const fieldError =
+      getIn(form.errors, fieldPath, null) ||
+      getIn(form.errors, 'pids', null) ||
```

Comment:
Isn't fieldPath set to `pids`?

Reply:
The fieldpath is set to `fieldPath={'pids.${pid.scheme}'}` ([see code here](https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposit/RDMDepositForm.js#L147)). I think we could remove the check for the hardcoded `pids` but not sure if this will cause other bugs as we are only using DOIs in the `pids` field. Maybe it's needed if other type of PIDs are registered? If you give the green light I will remove it

Reply:
I tried in our QA environment and I only see errors returned for the fieldPath rather than `pids` field.... I would say is safe to remove it now

Reply:
Ok, thanks! Done